docker-compose.yml

```diff
@@ -198,12 +198,13 @@ services:
     container_name: worker_site
       
   worker_compute:
-    image: codalab/competitions-v1-compute-worker:testing-realtime2
+    image: codalab/competitions-v1-nvidia-worker:reporter
```

Comment:
Are we pushing this to latest? May want to have this point there

Reply:
Good call